app/i18n/cz/sub.php

```diff
@@ -126,8 +126,8 @@
 			'reload_articles_help' => 'Znovu načíst články a získat úplný obsah, pokud je definován přepínač.',	// DIRTY
```

Comment:
Could you please check this line? It is marked as "Dirty". I am not sure why. Is it a correct translation of `Reload that many articles and fetch complete content if a selector is defined.` ?

Reply:
`přepínač` is `switch` IMO for selector is better translation `selektor`

Reply:
Do you want to change it or keep it as it is?

Reply:
Sorry for delay, i updated it after discuss with colleague